tools/run_in_docker/generate_rest_api_docs.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+# Copyright European Organization for Nuclear Research (CERN) since 2012
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from apispec import APISpec  # type: ignore
+from apispec_webframeworks.flask import FlaskPlugin  # type: ignore
+from rucio.vcsversion import VERSION_INFO  # type: ignore
+from rucio.web.rest.flaskapi.v1.main import application  # type: ignore
```

Comment:
what are the `# type: ignore` comments for?

Reply:
Maybe we could add them to the requirements instead - I guess they can be seen as part of the requirements for building the docs, so it makes sense that they'd need to be part of the `requirements.txt` file

```
  check_python_scripts:
    name: Check Python Scripts
    runs-on: ubuntu-latest
    steps:
      - name: Checkout repository
        uses: actions/checkout@v4
      - name: Install dependencies
        run: |
          pip install -r tools/requirements.txt
      - name: Run black
        run: |
          black --check .
      - name: Run isort
        run: |
          isort --profile black --filter-files --check .
      - name: Run flake8
        run: |
          flake8 --config tools/.flake8
      - name: MyPy
        run: |
          mypy --ignore-missing-imports .
```